discord/voice_state.py

```diff
@@ -459,13 +459,26 @@ async def soft_disconnect(self, *, with_state: ConnectionFlowState = ConnectionF
             if self.socket:
                 self.socket.close()
 
-    async def move_to(self, channel: Optional[abc.Snowflake]) -> None:
+    async def move_to(self, channel: Optional[abc.Snowflake], timeout: Optional[float]) -> None:
         if channel is None:
             await self.disconnect()
             return
 
-        await self.voice_client.channel.guild.change_voice_state(channel=channel)
-        self.state = ConnectionFlowState.set_guild_voice_state
+        # this is only an outgoing ws request; if it fails, nothing happens and nothing changes
+        await self._move_to(channel)
+
+        last_state = self.state
+        try:
+            await self.wait_async(timeout)
+        except asyncio.TimeoutError:
+            _log.warning(
+                'Timed out trying to move to channel %s in guild %s',
+                channel.id,
+                self.guild.id,
+            )

```

Comment:
Flatten this to one line please

Reply:
Black seems to want to expand this, do I make aliases or fmt: off or what?

Reply:
Remove the trailing comma.

Reply:
Ah yes I removed an arg but forgot the comma.